lib/logger/lib/logger.ex

```diff
@@ -395,7 +397,11 @@ defmodule Logger do
   @spec flush :: :ok
   def flush do
     _ = GenEvent.which_handlers(:error_logger)
-    _ = GenEvent.which_handlers(Logger)
+    handlers = GenEvent.which_handlers(Logger)
+    _ = for {Logger.Backends.Console, _} = handler <- handlers do
```

Comment:
I am not comfortable with hardcoding it. Maybe we should just send a message to logger so the backend can optionally handle it in `handle_info` and then we do a `which_handlers` call?